pkg/command/execution.go

```diff
@@ -0,0 +1,28 @@
+package command
+
+import (
+	"os/exec"
+	"sync"
+)
+
+type execution struct {
+	cmd           *exec.Cmd
+	wg            sync.WaitGroup
+	errCopyStdout error
+	errCopyStderr error
```

Comment:
Maybe add a comment here on why you needed to put this in the struct. At least it took me a little while to figure it out from the diff.

Reply:
done